example/src/main/java/org/nervos/ckb/SUDTExample.java

```diff
@@ -0,0 +1,196 @@
+    IndexerCollector txUtils = new IndexerCollector(api, ckbIndexerApi);
+
+    List<Receiver> receivers =
+        Collections.singletonList(new Receiver(SendAddresses.get(0), Utils.ckbToShannon(400)));
+    List<CellOutput> cellOutputs = txUtils.generateOutputs(receivers, SendAddresses.get(0));
+    cellOutputs.get(0).type = sudtType;
+    txBuilder.addOutputs(cellOutputs);
+
+    txBuilder.setOutputsData(
+        Arrays.asList(Numeric.toHexString(new UInt128(SUDT_ISSUE_SUM_AMOUNT).toBytes()), "0x"));
+    txBuilder.addCellDep(new CellDep(new OutPoint(SUDT_TX_HASH, "0x0"), CellDep.CODE));
+
+    // You can get fee rate by rpc or set a simple number
+    BigInteger feeRate = BigInteger.valueOf(1024);
+
+    // initial_length = 2 * secp256k1_signature_byte.length
+    // collectInputsWithIndexer method uses indexer rpc to collect cells quickly
+    CollectResult collectResult =
+        txUtils.collectInputs(SendAddresses, txBuilder.buildTx(), feeRate, Sign.SIGN_LENGTH * 2);
+
+    // update change cell output capacity after collecting cells if there is changeOutput
+    if (Numeric.toBigInt(collectResult.changeCapacity).compareTo(MIN_CKB) >= 0) {
+      cellOutputs.get(cellOutputs.size() - 1).capacity = collectResult.changeCapacity;
+      txBuilder.setOutputs(cellOutputs);
+    }
+
+    int startIndex = 0;
+    for (CellsWithAddress cellsWithAddress : collectResult.cellsWithAddresses) {
+      txBuilder.addInputs(cellsWithAddress.inputs);
+      for (int i = 0; i < cellsWithAddress.inputs.size(); i++) {
+        txBuilder.addWitness(i == 0 ? new Witness(Witness.SIGNATURE_PLACEHOLDER) : "0x");
+      }
+      if (cellsWithAddress.inputs.size() > 0) {
+        scriptGroupWithPrivateKeysList.add(
+            new ScriptGroupWithPrivateKeys(
+                new ScriptGroup(
+                    NumberUtils.regionToList(startIndex, cellsWithAddress.inputs.size())),
+                Collections.singletonList(
+                    SendPrivateKeys.get(SendAddresses.indexOf(cellsWithAddress.address)))));
+        startIndex += cellsWithAddress.inputs.size();
+      }
+    }
+
+    Secp256k1SighashAllBuilder signBuilder = new Secp256k1SighashAllBuilder(txBuilder.buildTx());
+
+    for (ScriptGroupWithPrivateKeys scriptGroupWithPrivateKeys : scriptGroupWithPrivateKeysList) {
+      signBuilder.sign(
+          scriptGroupWithPrivateKeys.scriptGroup, scriptGroupWithPrivateKeys.privateKeys.get(0));
+    }
+    Transaction tx = signBuilder.buildTx();
+    return api.sendTransaction(tx);
+  }
+
+  private static String transfer() throws IOException {
+    List<ScriptGroupWithPrivateKeys> scriptGroupWithPrivateKeysList = new ArrayList<>();
+
+    TransactionBuilder txBuilder = new TransactionBuilder(api);
+    IndexerCollector txUtils = new IndexerCollector(api, ckbIndexerApi);
+
+    List<Receiver> receivers =
+        Collections.singletonList(new Receiver(ReceiveAddresses.get(0), Utils.ckbToShannon(150)));
```

Comment:
Would it be better to use 142 here?

Reply:
Why？I think this example is not used to illustrate the minimum capacity of sudt.

Reply:
no more than 142 in real scenarios.

Reply:
Updated in https://github.com/nervosnetwork/ckb-sdk-java/pull/381/commits/14b9fe5e0b76be7f7983287f122c9d4f6bb58118